tests/mr/device/mr_multithreaded_tests.cpp

```diff
@@ -179,6 +179,7 @@ void allocate_loop(rmm::mr::device_memory_resource* mr,
                    std::size_t num_allocations,
                    std::list<allocation>& allocations,
                    std::mutex& mtx,
+                   std::condition_variable& cv,
```

Comment:
Pretty sure clang-tidy will complain about this name being less than 3 characters. Suggest naming it after the condition that it signals. Suggest 'allocations_ready'.

Reply:
updated to use the name `allocations_ready`